src/main/java/com/knighttodo/knighttodo/rest/TodoResource.java

```diff
@@ -67,12 +68,12 @@
         return ResponseEntity.status(HttpStatus.FOUND).body(todoRestMapper.toTodoResponseDto(todoVO));
     }
 
-    @PutMapping
-    public ResponseEntity<UpdateTodoResponseDto> updateTodo(@Valid @RequestBody UpdateTodoRequestDto updateRequestDto) {
+    @PutMapping("/{todoId}")
```

Comment:
I think we need to rename all same path variables in other methods to {todoId} like here instead of {id}

Reply:
yes, you're right, didn't want to add more things to this MR.

Reply:
okay)